### PR TITLE
change customer password from admin

### PR DESCRIPTION
**Pull request by priyankavasoya**
@harishmk1805

### CODE REVIEW

File: app/code/Dss/ChangeCustomerPassword/Command/CustomerChangePasswordCommand.php [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+     * @param CustomerResource $resource
+     * @param AppState $state
+     */
+    public function __construct(
+        private CustomerFactory $customerFactory,
+        private StoreManagerInterface $storeManager,
+        private CustomerResource $resource,
+        private AppState $state
+    ) {
+        parent::__construct();
```
Comment (harishmk1805) [inline app/code/Dss/ChangeCustomerPassword/Command/CustomerChangePasswordCommand.php:50]: remove parent::__construct();

File: app/code/Dss/ChangeCustomerPassword/composer.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    ],
+    "license": [
+      "OSL-3.0",
+      "AFL-3.0"
+    ],
+    "require": {
+      "php": "^8.1",
+      "magento/product-community-edition": "^2.4.5",
+      "dss/m2-core": "^0.0.1",
+      "dss/m2-seo-core": "^0.0.1"
```
Comment (harishmk1805) [inline app/code/Dss/ChangeCustomerPassword/composer.json:21]: where it's used "dss/m2-seo-core": "^0.0.1"  ??

**Review by harishmk1805 — CHANGES_REQUESTED** [review]
please resolve this changes

File: app/code/Dss/ChangeCustomerPassword/Command/CustomerChangePasswordCommand.php (hunk 1 above)
Comment (priyankavasoya) [reply app/code/Dss/ChangeCustomerPassword/Command/CustomerChangePasswordCommand.php]: done

File: app/code/Dss/ChangeCustomerPassword/composer.json (hunk 2 above)
Comment (priyankavasoya) [reply app/code/Dss/ChangeCustomerPassword/composer.json]: not used this is my mistake removed

File: app/code/Dss/ChangeCustomerPassword/Command/CustomerChangePasswordCommand.php (hunk 1 above)
Comment (harishmk1805) [reply app/code/Dss/ChangeCustomerPassword/Command/CustomerChangePasswordCommand.php:50]: use?

**Review by harishmk1805 — APPROVED** [review]

**Pull request merged**